helm/primehub/templates/secret.yaml

```diff
@@ -0,0 +1,12 @@
+# This is a place holder of primehub-secret
```

Comment:
let's rephrase the comments:
> The secret data will be populated by bootstrap job.
> Note that `helm diff` will not show the data content, since they are considered local changes

Reply:
Resovled
https://github.com/InfuseAI/primehub/blob/24a1acf7cb64171319a2631f40572ea8aec3a304/helm/primehub/templates/secret.yaml#L1-L3